### PR TITLE
fix(frontend): Add hedge cases to util `parseIcErrorMessage`

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

We treat more hedge cases in util `parseIcErrorMessage` and create tests to check.

### CODE REVIEW

File: src/frontend/src/lib/utils/error.utils.ts [hunk 1]
```diff
@@ -102,7 +102,7 @@ export const replaceIcErrorFields = (err: unknown): string | undefined =>
 
 const stripHttpDetails = (text: string): string =>
 	text
-		// Remove from "HTTP d'-0tails:" + "{" up to the next closing brace on its own line
+		// Remove from "HTTP details:" + "{" up to the next closing brace on its own line
```
Comment (roman-nazaruk) [inline src/frontend/src/lib/utils/error.utils.ts:105]: Wow, thanks
